config/deploy.rb

```diff
@@ -29,7 +29,7 @@
 # set :pty, true
 
 # Default value for :linked_files is []
-set :linked_files, %w{bin/write_marc_record}
+# set :linked_files, []
```

Comment:
we could, but i like leaving the options there, but commented out, so you know what they are easily for potential future use (that is true of the other unused options in the deploy script too)